tests/NuGet.Services.AzureSearch.Tests/SearchDocumentBuilderFacts.cs

```diff
@@ -824,6 +932,59 @@ public abstract class BaseFacts
                 new object[] { SearchFilters.IncludePrereleaseAndSemVer2, "IncludePrereleaseAndSemVer2" },
             };
 
+            public static IEnumerable<object[]> PackageTypesData => new[]
+            {
+                new object[] {
+                    new List<PackageType> {
+                        new PackageType
+                        {
+                            Name = "DotNetCliTool"
+                        }
+                    },
+                    @"""dotnetclitool""",
```

Comment:
Note DotnetTool is more the canonical example

Reply:
Can we cover both the null and empty array cases?

Reply:
There is null case and empty array case below. Or do you mean when the page will have null/empty? This should never be the case.

Reply:
Oh sorry, I missed it. Looks great.